crates/relayer/src/channel.rs

```diff
@@ -1144,16 +1229,17 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> Channel<ChainA, ChainB> {
             ));
         }
 
-        // Connection must exist on destination
-        self.dst_chain()
-            .query_connection(
-                QueryConnectionRequest {
-                    connection_id: self.dst_connection_id().clone(),
-                    height: QueryHeight::Latest,
-                },
-                IncludeProof::No,
-            )
-            .map_err(|e| ChannelError::query(self.dst_chain().id(), e))?;
+        // Connection must exist on destination -- THIS WAS ALREADY CHECKED ON fn run(), if we are not storing
```

Comment:
It just occurred to me that the reason why this check exists in here as well, even though it is also present in `fn run()` for `TxChanOpenTryCmd`, is because the `build_chan_open_try()` method may be called from somewhere other than the CLI functions, in which case this check may not have been performed yet, correct me if I'm wrong. In that case we should keep this.

Reply:
Yes exactly, so let's keep it as a sanity check.